iroh-relay/src/node_info.rs

```diff
@@ -394,23 +401,24 @@ impl<T: FromStr + Display + Hash + Ord> TxtAttrs<T> {
     }
 
     /// Looks up attributes by DNS name.
-    pub(super) async fn lookup_by_name(resolver: &DnsResolver, name: &str) -> Result<Self> {
+    #[cfg(not(wasm_browser))]
+    pub(crate) async fn lookup_by_name(resolver: &DnsResolver, name: &str) -> Result<Self> {
         let name = Name::from_str(name)?;
         TxtAttrs::lookup(resolver, name).await
     }
 
     /// Returns the parsed attributes.
-    pub(super) fn attrs(&self) -> &BTreeMap<T, Vec<String>> {
+    pub(crate) fn attrs(&self) -> &BTreeMap<T, Vec<String>> {
         &self.attrs
     }
 
     /// Returns the node id.
-    pub(super) fn node_id(&self) -> NodeId {
+    pub(crate) fn node_id(&self) -> NodeId {
         self.node_id
     }
 
     /// Parses a [`pkarr::SignedPacket`].
-    pub(super) fn from_pkarr_signed_packet(packet: &pkarr::SignedPacket) -> Result<Self> {
+    pub(crate) fn from_pkarr_signed_packet(packet: &pkarr::SignedPacket) -> Result<Self> {
```

Comment:
can we do pkarr based lookups in the browser? are we doing that already?

Reply:
Yes we can! There's a basic HTTP-based route on the dns server we have (`/relay`).
See also `discovery_n0` in the builder - and how we replace `DnsDiscovery` with `PkarrResolver` in the browser.

Reply:
lovely